### ISSUE TITLE
[Menu] Screen reader does not announce keyboard shortcuts in menu options

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#2665]
### Provide a general summary of the issue here

**Description:**
For the variant with keyboard shortcuts, screen reader users are currently not informed that part of the read text refers to a keyboard shortcut. For example, in the menu, options like “List Ctrl1”, “Cards Ctrl2”, and “Grid Ctrl3” are visually clear, but screen readers read them as plain text without indicating that “Ctrl1”, “Ctrl2”, etc., are keyboard shortcuts.

<img width="1132" height="640" alt="Image" src="https://github.com/user-attachments/assets/74c3f109-3ffb-470d-926b-c2c8d88358f1" />

**Expected behavior:**
Screen reader users should be informed when part of the label refers to a keyboard shortcut. This can be achieved by adding a visually hidden label (e.g., using aria-label, aria-describedby, or a visually hidden span) to clarify that “Ctrl1” is a keyboard shortcut.

**Link to finding:** https://github.com/sl-design-system/components/issues/2574#issuecomment-3298429665

### 👤 Your name

DR

### 🧢 Your Product/Team

SLDS
